qutebrowser/app.py

```diff
@@ -210,14 +210,17 @@ def _load_session(name):
         name: The name of the session to load, or None to read state file.
     """
     state_config = objreg.get('state-config')
+    session_manager = objreg.get('session-manager')
     if name is None:
         try:
             name = state_config['general']['session']
         except KeyError:
             # No session given as argument and none in the session file ->
             # start without loading a session
-            return
-    session_manager = objreg.get('session-manager')
+            if session_manager.exists('_autosave'):
```

Comment:
Here, I've been giving priority to the session saved by the user. This means, that if the user quits with `:wq`, then restarts `qutebrowser`, and then `qutebrowser` crashes for some reason, next time `qutebrowser` is restarted, the session saved lastly with `wq` will be saved, and not the one autosaved.

What do you think?